site/docs/reference/connectors-datacatalog/Apis/DefaultApi.md

```diff
@@ -5,6 +5,7 @@ All URIs are relative to *https://localhost:8080*
 Method | HTTP request | Description
 ------------- | ------------- | -------------
 [**createAsset**](DefaultApi.md#createAsset) | **POST** /createAsset | This REST API writes data asset information to the data catalog configured in fybrik
+[**deleteAsset**](DefaultApi.md#deleteAsset) | **POST** /deleteAsset | This REST API deletes data asset from the data catalog configured in fybrik
```

Comment:
I'd change it to just " **deletes** data asset ... " it is relevant to all calls

Reply:
Fixed. Thanks.